src/js/view/view.js

```diff
@@ -401,6 +402,7 @@ define([
 
             _videoLayer = _playerElement.getElementsByClassName('jw-media')[0];
 
+            _mediaLayer = _playerElement.querySelector('.jw-media');
```

Comment:
What is the difference between `_videoLayer` and `_mediaLayer`? It's the same element:

`getElementsByClassName('jw-media')[0] === querySelector('.jw-media')`